sqs.log

```diff
@@ -1,665 +1,1472 @@
-{"level":"info","ts":1713407057.397006,"msg":"log level","is_debug":false,"log_level":"info"}
```

Comment:
nit: do you mind deleting this file before final review? 🙏

Reply:
noted